pyvrp/tests/test_TimeWindowSegment.py

```diff
@@ -32,7 +32,7 @@ def test_merge_multiple():
     tws2 = TimeWindowSegment(1, 1, 0, 0, 3, 6)
     tws3 = TimeWindowSegment(2, 2, 0, 0, 2, 3)
 
-    mat = IntMatrix([[1, 4, 1], [1, 2, 4], [1, 1, 1]])
+    mat = DoubleMatrix([[1, 4, 1], [1, 2, 4], [1, 1, 1]])
```

Comment:
This does not pass when compiling in integer mode, but I'm OK with that. Once we implement #61, the issue is moot anyway.

Reply:
I'm fine with that, however could we somehow make a way to inspect from Python whether PyVRP was compiled in integer or double precision mode? That may be helpful to the user and then we could also easily make this test work.

Reply:
It's already possible via signature inspection at runtime; that's no problem. But the larger question is how to make these different compilation modes available to the user, and I don't really have ideas about that yet.